stackanetes/common/utils.py

```diff
@@ -21,3 +25,35 @@ def env(*args, **kwargs):
         if value:
             return value
     return kwargs.get('default', '')
+
+
+def get_kubectl_command(add_namespace=True):
+    cmd = [CONF.stackanetes.kubectl_path]
+    if CONF.stackanetes.host:
+        server = "--server=" + CONF.stackanetes.host
```

Comment:
What about white spaces after each additional parameter? We'll end up with:
--server=abc--kubeconfig=configpath